sklearn/metrics/_plot/roc_curve.py

```diff
@@ -75,43 +130,61 @@ class RocCurveDisplay(_BinaryClassifierCurveDisplayMixin):
     >>> pred = np.array([0.1, 0.4, 0.35, 0.8])
     >>> fpr, tpr, thresholds = metrics.roc_curve(y, pred)
```

Comment:
Do `fpr`and `tpr` change to plural (`fprs`and `tprs`) only after they are officially deprecated?

Reply:
The singular parameters will be deprecated in 1.7, and then completely removed in 1.9. Between this time users will still be able to use the old singular parameter but they will be given a deprecation warning.

Bit hard to find, but some more details about deprecation here: https://scikit-learn.org/dev/developers/contributing.html#deprecation

Reply:
Thank you for the link to deprecation. 
What I wanted to say was that a new user will see that `fpr` and `tpr` are deprecated (warnings and descriptions in the docs).. But then see them being used or included in the example. 
The deprecation link doesn't specify what to use in the examples I think.
 
It's truly a "microscopic" detail. We can discuss it during the call if you think we should. Otherwise you can just ignore me. 🙂

Reply:
> What I wanted to say was that a new user will see that fpr and tpr are deprecated (warnings and descriptions in the docs).. But then see them being used or included in the example.

We actually check that the examples don't raise future warnings so we need to replace them by the new names in the doc. Pushing a commit with `[doc build]` will run all the examples and show all occurrences to update. But it's not urgent, it can be done in a later stage of the PR when all the names and API details are final.

Reply:
Oh I see what you are referring to now.

I do use the new `RocCurveDisplay` parameters (the doc testing did run the docstring examples):

```
    >>> display = metrics.RocCurveDisplay(fprs=[fpr], tprs=[tpr], roc_aucs=[roc_auc],
    ...                                   names=['example estimator'])
```

but in the above line:

```
fpr, tpr, thresholds = metrics.roc_curve(y, pred)
```
we do only get one set of `fpr` and `tpr` out so I thought I would leave these are singular. Open to change though.